src/frontend/src/app/pages/open-proposal-details/open-proposal-details.component.ts

```diff
@@ -0,0 +1,214 @@
+import { CommonModule, DatePipe } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  OnInit,
+  SecurityContext,
+} from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { DomSanitizer } from '@angular/platform-browser';
+import { ActivatedRoute, Router } from '@angular/router';
+import { marked } from 'marked';
+import { filter, map } from 'rxjs';
+
+import { ProposalLinkBaseUrl, ProposalState, ProposalTopic } from '~core/state';
+import { ProposalService } from '~core/state';
+import {
+  CardComponent,
+  KeyColComponent,
+  KeyValueGridComponent,
+  ValueColComponent,
+} from '~core/ui';
+import { isNotNil } from '~core/utils';
+
+@Component({
+  selector: 'app-open-proposal-details',
+  standalone: true,
+  imports: [
+    CommonModule,
+    CardComponent,
+    KeyValueGridComponent,
+    KeyColComponent,
+    ValueColComponent,
+    DatePipe,
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styles: [
+    `
+      @import '@cg/styles/common';
+
+      .proposal {
+        margin-bottom: size(6);
+      }
+
+      .proposal-title {
+        word-break: break-all;
+      }
+
+      :host {
```

Comment:
We should make a shared css class/component to share with other pages: https://github.com/CodeGov-org/codegov-website/blob/main/src/frontend/src/app/pages/profile-edit/profile-edit.component.ts#L32-L44